# control commands for the car
std_msgs/Header header
float64 steer_angle     # steer angle in degrees
float64 throttle        # throttle, range 0.0-1.0, may add negative value for brake in future
